.github/workflows/dependabot-batch-updater.yml

```
@@ -0,0 +1,19 @@
+name: Check if dependencies can be updated and create a pull request with updated ones.
+on:
+  schedule:
+    # Run once a week at 5:00
+    - cron:  '0 5 * * */7'
+
+jobs:
+  update_dependencies:
+    runs-on: ubuntu-20.04
+    name: Update dependencies
+    steps:
+      - uses: rasaHQ/dependabot-batch-updater@main
```

```suggestion
      - uses: RasaHQ/dependabot-batch-updater@main
```
Can we use a tag instead of the `main` branch? I'd start with `v1.0`